src/LoginProviders/LoginProvider.php

```diff
@@ -82,15 +82,15 @@ protected function detectRememberMe()
         if (!$this->isLoggedIn()) {
             $request = Request::current();
 
-            if ($request->cookie('lun') != "") {
```

Comment:
these are magic getter methods. if you call a method which doesn't exist on a request object, it looks for a key of the first param in an array called {methodName}Data on the request object. So this shouldn't need to be changed at all